resources/markdown/who_audit_l2.md

```diff
@@ -0,0 +1,282 @@
+![Workflow](https://github.com/projectcypress/promoting-interoperability-test-artifacts/blob/867ceae3e7604fd3a457bd1fad702c5ce0ce1f39/exports/who_data/WHO_AUDIT_L2.png)
+--------------------------------------------------------------------------------------------------------------------------------------------
+# STEP 1: Consider Alcohol Pre-Screen
+
+### Inclusion:
+<pre>
+Patient is >= 18 years old
+</pre>
+
+### Exclusion:
```

Comment:
Alcohol screening will be defined as: 
USAUDIT-C score (rep. by a local code)
OR USAUDIT score (rep. by a local code)
OR (WHO) AUDIT-C score (rep. by a VS that includes the LOINC code for this score)
OR (WHO) AUDIT score (rep. by a VS that includes the LOINC code for this score)
OR APS response = “No” (rep. by a local code)
OR APS response of “Yes” (rep. by a local code)
     AND 
          USAUDIT Q1 score = “0”  (rep. by a local code)
          OR (WHO) Q1 score = “0”  (rep. by a local code)